website/docs/docs/cloud-integrations/downstream-exposures-tableau.md

```diff
@@ -0,0 +1,107 @@
+   <Lightbox src="/img/docs/cloud-integrations/auto-exposures/tablueau-serverurl.jpg" title="Locate the Server URL and Sitename in Tableau"/>
+
+   For example, if the full URL is: `10az.online.tableau.com/#/site/dbtlabspartner/explore`:
+   - The **Server URL** is the first part of the URL, in this case: `10az.online.tableau.com`
+   - The **Sitename** is right after the `site` in the URL, in this case: `dbtlabspartner` 
+
+5. You should now be ready to set up downstream exposures in dbt Cloud after copying the following items, which you'll need during the dbt Cloud setup: 
+      - ServerURL
+      - Sitename
+      - Token name
+      - Secret
+
+### Set up in dbt Cloud
+
+1. In dbt Cloud, navigate to the project you want to add the downstream exposure to and then select **Settings**.
+2. Under the **Exposures** section, select **Add integration** to add the Tableau connection.
+   <Lightbox src="/img/docs/cloud-integrations/auto-exposures/cloud-add-integration.jpg" title="Select Add Integration to add the Tableau connection."/>
+3. Enter the details for the exposure connection you collected from Tableau in the [previous step](#set-up-in-tableau) and click **Continue**. Note that all fields are case-sensitive.
+   <Lightbox src="/img/docs/cloud-integrations/auto-exposures/cloud-integration-details.jpg" title="Enter the details for the exposure connection."/>
+4. Select the collections you want to include for the downstream exposures and click **Save**.
+   
+   <Lightbox src="/img/docs/cloud-integrations/auto-exposures/cloud-select-collections.jpg" title="Select the collections you want to include for the downstream exposures."/>
+
+      :::info
+      dbt Cloud automatically imports and syncs any workbook within the selected collections. New additions to the collections will be added to the lineage in dbt Cloud during the next sync (automatically once per day).
+   
+      dbt Cloud immediately starts a sync when you update the selected collections list, capturing new workbooks and removing irrelevant ones.
+      :::
+
+5. dbt Cloud imports everything in the collection(s) and you can continue to [view them](#view-auto-exposures) in Explorer. 
+
+   <Lightbox src="/img/docs/cloud-integrations/auto-exposures/explorer-lineage2.jpg" width="100%" title="View from the dbt Explorer in your Project lineage view, displayed with the Tableau icon."/>
+
+import ViewExposures from '/snippets/_auto-exposures-view.md';
+
+<ViewExposures/>
+
+## Orchestrate exposures <Lifecycle status="beta"/>
+
+[Orchestrate exposures](/docs/cloud-integrations/orchestrate-exposures) using the dbt [Cloud job scheduler](/docs/deploy/job-scheduler) to proactively refresh the underlying data sources (extracts) that power your Tableau Workbooks.
+
+- Running exposures with a job run integrates with downstream exposures and uses your `dbt build` job to ensure that Tableau extracts are updated regularly and automatically.
```

Comment:
This sentence is a little hazy to me. We're saying "job run" and then "dbt build job" and I'm not clear on what integrates means in this scenario. Will this be apparent to dbt and Tableau users or do we need to add a little context?

Reply:
good shout, have changed it to below:

> Orchestrating exposures with a `dbt build` job ensures that downstream exposures, like Tableau extracts, are updated regularly and automatically.